psi4/driver/procrouting/sapt/sapt_proc.py

```diff
@@ -392,7 +392,9 @@ def sapt_dft(dimer_wfn, wfn_A, wfn_B, sapt_jk=None, sapt_jk_B=None, data=None, p
     core.print_out("\n")
     aux_basis = core.BasisSet.build(dimer_wfn.molecule(), "DF_BASIS_MP2",
                                     core.get_option("DFMP2", "DF_BASIS_MP2"), "RIFIT", core.get_global_option('BASIS'))
-    is_hybrid = wfn_B.functional().is_x_hybrid()
+    is_hybrid = core.get_option("SAPT", "SAPT_DFT_DO_HYBRID")
+    if not wfn_B.functional().is_x_hybrid() or wfn_B.functional().is_x_lrc():
+        is_hybrid = False
```

Comment:
you'll want to think through all the cases, as in the table below for just x_hybrid. there's a scenario where you're overriding the deliberate settings of the user w/o error or warning. can use `core.has_option_changed('SAPT', 'SAPT_DFT_DO_HYBRID')` to see if user has touched option.

user | x_hybrid | wanted result | current code result
-----|------|---|---
does nothing | T | hybrid | hybrid
sets True | T | hybrid | hybrid
sets  False | T | no hybrid | no hybrid 
does nothing | F | no hybrid | no hybrid
sets True | F | error msg | no hybrid
sets  False | F | no hybrid | no hybrid

For LRC, I think you want something like the below.

user | gga fctl | hyb fctl | lrc fctl
---|---|---|---
does nothing | no hyb disp | hyb disp | NYI error
sets True | incompatible error | hyb disp | NYI error
sets False | no hyb disp | no hyb disp | no hyb disp

Reply:
The logic of the bottom table is now implemented except "lrc"+"does nothing" is a printed warning. Leaving conversation unresolved so other reviewers can object to the table.